src/pages/MovieDetails.vue

```diff
@@ -15,44 +36,211 @@ export default defineComponent({
   },
   data() {
     return {
-      movieDetails: null,
-      isLoading: true,
-      errorMessage: null,
+      movie: null,
+      movieIsLoading: true,
+      movieError: null,
+      date: new Date(),
     };
   },
   computed: {
+    ...mapState(useSeancesStore, [
+      "seances",
+      "seancesIsLoading",
+      "seancesError",
+      "getSeancesErrorMessage",
+    ]),
+    getMovieErrorMessage() {
+      return (
+        this.movieError?.message ||
+        "We are sorry, but the movie cannot be displayed"
+      );
+    },
     getMovieTitle() {
-      return this.movieDetails.title;
+      return this.movie.title;
+    },
+    getMovieReleaseYear() {
```

Comment:
you could do the same for those computed properties getting things from store, what do you think? 💡